<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="s3.ind.business.impl.WorkoutServiceImplTest" tests="5" skipped="0" failures="0" errors="0" timestamp="2024-05-06T18:48:44" hostname="DESKTOP-88OFGPG" time="0.046">
  <properties/>
  <testcase name="createWorkout_WhenTitleExists_ThrowsException()" classname="s3.ind.business.impl.WorkoutServiceImplTest" time="0.011"/>
  <testcase name="createWorkout_Successful_ReturnsResponse()" classname="s3.ind.business.impl.WorkoutServiceImplTest" time="0.008"/>
  <testcase name="deleteWorkout_Successful_DeletesWorkout()" classname="s3.ind.business.impl.WorkoutServiceImplTest" time="0.005"/>
  <testcase name="updateWorkout_WorkoutNotFound_ThrowsException()" classname="s3.ind.business.impl.WorkoutServiceImplTest" time="0.007"/>
  <testcase name="updateWorkout_Successful_UpdatesFields()" classname="s3.ind.business.impl.WorkoutServiceImplTest" time="0.004"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
